### ISSUE TITLE
Pinch Zoom On/Off button not present in 2.8.0.98

### ISSUE THREAD

**Issue by oli3012** [ddvk/remarkable-hacks#332]
Tested on a rm1, the button is not present.

**Comment by ddvk** [comment]
as it was software default now and not an extra feature it was removed. do you happen to activate it unintentionally?

**Comment by oli3012** [comment]
Yes I often activate it unintentionally. I bought the RM1 precisely because
there is physical buttons for navigation. But now that they added touch I
need to be very careful on how I manipulate the device to avoid triggering
it. I wish there was a way to disable it.
